journeys/shourgamer2.md

```diff
@@ -0,0 +1,11 @@
+## How I Began Coding!
+I started coding in 2020. It was lockdown and it was boring to play games all day so I decided to learn to code. My first language I bealive was python or C#. 
+### Shourgamer2's Journey
+I was intrested in making games so I learnt the [unity engine](https://unity.com/) and started using it. I used to code all day at that time and I was working on a project called the "Golden pistol"
+Which was a fps story game. Sadly after I exported the game I didn't publish it. Well then I got intrest in roblox games (which I have tried before but at that time I thought it will be a waste of time to make games in roblox)
```

Comment:
```
Well then I got intrest in roblox games (which I have tried before but at that time I thought it will be a waste of time to make games in roblox)
```

can you link your roblox dev page?

Reply:
If you don't have a roblox dev page then no worries, If you link the page, then people can see what games you've made!